### PR TITLE
Add support for downloading sales invoices

### PR DESCRIPTION
**Pull request by kerim1510**
Support downloading sales invoices via /sales_invoices/:id/download_pdf.

This PR also fixes issue #46.

### CODE REVIEW

**Comment by kerim1510** [comment]
@maartenvanvliet can you review this PR, our production environment is depending on it.

File: lib/moneybird/service/sales_invoice.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@ class SalesInvoice
     include Moneybird::Traits::Save
     include Moneybird::Traits::Delete
     include Moneybird::Traits::Synchronization
+    include Moneybird::Traits::DownloadPdf
```
Comment (maartenvanvliet) [inline lib/moneybird/service/sales_invoice.rb:10]: Is this trait a shared by other invoice types/resources on the moneybird side?

File: lib/moneybird/traits/download_pdf.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+module Moneybird
+  module Traits
+    module DownloadPdf
+      def download_pdf(id)
```
Comment (maartenvanvliet) [inline lib/moneybird/traits/download_pdf.rb:4]: Could you add a test regarding the following of the 302 redirect?

**Review by maartenvanvliet — CHANGES_REQUESTED** [review]
Looks good overall, two questions

File: lib/moneybird/service/sales_invoice.rb (hunk 1 above)
Comment (kerim1510) [reply lib/moneybird/service/sales_invoice.rb:10]: It could also be use at estimates. But this PR intended to cover only the download option at invoices.

Comment (maartenvanvliet) [reply lib/moneybird/service/sales_invoice.rb:10]: Ah cool, good to know.

**Review by maartenvanvliet — APPROVED** [review]
Great thanks!!

**Pull request merged**